contrib/docs-examples/snippets/react-native/StitchAuth.loginWithCredential.js

```diff
@@ -0,0 +1,98 @@
+function example() {} function untested({View, Text, Button, React, GoogleSignin, Stitch, GoogleCredential, GoogleSigninButton}) { // too much boilerplate to be able to test this
```

Comment:
I get the reasoning for this change, but it seems incomplete to not include the anonymous and email/password login snippets here too. Maybe later down the road we can look into some kind of tabbed experience in the SDK docs to pick the credential you want an example for so that we can include them here as well as in StitchAuth?

Reply:
Don't worry, those snippets are still available elsewhere in the docs!